builder/dockerfile/parser/parser.go

```diff
@@ -331,8 +335,12 @@ func trimWhitespace(src []byte) []byte {
 	return bytes.TrimLeftFunc(src, unicode.IsSpace)
 }
 
+func isComment(line []byte) bool {
+	return bytes.HasPrefix(trimWhitespace(line), []byte("#"))
```

Comment:
Do you think this ought to re-use `tokenComment` to keep a single-source-of-truth, or maybe it doesn't matter?

Reply:
oh, good one, yes, can do